documentation/missionmaker/classnames.md

```diff
@@ -0,0 +1,178 @@
+ACE_HandFlare_Yellow | M127A1 Hand Held Signal (Yellow)
+ACE_M84 | M84 Stun Grenade
+
+### hearing
+
+classname | in game name
+--------- | ---------
+ACE_EarPlugs | Ear Plugs
+
+### Kestrel
+
+classname | in game name
+--------- | ---------
+ACE_Kestrel4500 | Kestrel 4500NV
+
+### Laser pointers
+
+classname | in game name
+--------- | ---------
+acc_pointer_IR | Laser Pointer (red)
+ACE_acc_pointer_red | Laser Pointer (green)
+ACE_acc_pointer_green_IR | Laser Pointer (green)
+ACE_acc_pointer_green | Laser Pointer (green)
+
+### logistics
+
+classname | in game name
+--------- | ---------
+ACE_UAVBattery | UAV Battery
+ACE_wirecutter | Wirecutter
+
+### Map tools
+
+classname | in game name
+--------- | ---------
+ACE_MapTools | Map Tools
+
+# Medical
+
+classname | in game name
+--------- | ---------
+ACE_atropine | Atropin autoinjector
+ACE_fieldDressing | Bandage (Basic)
+ACE_elasticBandage | Bandage (Elastic)
+ACE_quikclot | Basic Field Dressing (QuikClot)
+ACE_bloodIV | Blood IV (1000ml)
+ACE_bloodIV_500 | Blood IV (500ml)
+ACE_bloodIV_250 | Blood IV (250ml)
+ACE_bodyBag | Bodybag
+ACE_epinephrine | Epinephrine autoinjector
+ACE_morphine | Morphine autoinjector
+ACE_packingBandage | Packing Bandage
+ACE_personalAidKit | Personal Aid Kit
+ACE_plasmaIV | Plasma IV (1000ml)
+ACE_plasmaIV_500 | Plasma IV (500ml)
+ACE_plasmaIV_250 | Plasma IV (250ml)
+ACE_salineIV | Saline IV (1000ml)
+ACE_salineIV_500 | Saline IV (500ml)
+ACE_salineIV_250 | Saline IV (250ml)
+ACE_surgicalKit | Surgical Kit
+ACE_tourniquet | Tourniquet (CAT)
+
+### Microdagr
+
+classname | in game name
+--------- | ---------
+ACE_microDAGR | MicroDAGR GPS
+
+classname | in game name
+--------- | ---------
+ACE_RangeTable_82mm | 82mm Rangetable
+
+### Nvgs
+classname | in game name
+--------- | ---------
+ACE_NVG_Gen1 | NV Goggles (Gen1)
+ACE_NVG_Gen2 | NV Goggles (Gen2)
+ACE_NVG_Gen3 | NV Goggles (Gen3)
+ACE_NVG_Gen4 | NV Goggles (Gen4)
+ACE_NVG_Wide | NV Goggles (Wide)
+
+
+### Optics
+classname | in game name
+--------- | ---------
+ACE_optic_Arco_2D | ARCO (2D)
+ACE_optic_Arco_PIP | ARCO (PIP)
+ACE_optic_LRPS_2D | LRPS (2D)
+ACE_optic_LRPS_PIP | LRPS (PIP)
+ACE_optic_SOS_2D| MOS (2D)
+ACE_optic_SOS_PIP | MOS (PIP
+ACE_optic_MRCO_2D | MRCO (2D)
+ACE_optic_MRCO_PIP | MRCO (PIP)
+ACE_optic_Hamr_2D | RCO (2D)
+ACE_optic_Hamr_PIP | RCO (PIP)
+
+### Overheating
+classname | in game name
+--------- | ---------
+ACE_SpareBarrel | Spare barrel
+
+### Parachute
+
+classname | in game name
+--------- | ---------
+ACE_Altimeter | Altimeter Watch
+ACE_NonSteerableParachute | Non-Steerable Parachute
+
+### Vector
+classname | in game name
+--------- | ---------
+ACE_Vector | Vector 21
+
+### Vehicle lock
+
+classname | in game name
+--------- | ---------
+ACE_key_master | Vehicle Key: Master
+ACE_key_lockpick | Lockpick
+ACE_key_west | Vehicle Key: West
+ACE_key_east | Vehicle Key: East
+ACE_key_indp | Vehicle Key: Independent
+ACE_key_civ | Vehicle Key: Civilian
```

Comment:
Classname list could do with saying if the classname is a weapon, item, magazine, etc.